CHANGELOG.rst

```diff
@@ -1,6 +1,42 @@
 CHANGELOG
 =========
 
+0.2.0 (2024-08-29)
+------------------
+
+**Breaking changes**
+
+- We removed the "use_gpu" input parameter from the SetupRayCluster and DeleteRayCluster operators. GPU drivers get installed if GPU nodes are available
+- Spelling correction in the ``SubmitRayJob``operator. Changed "self.terminal _state" to "self.terminal_states"
+
+**Enhancements to the SubmitRayJob operator**
```

Comment:
Could you please add the link to PR as well?

Reply:
added